app/editor/src/features/admin/reports/utils/openPreviewInNewTab.ts

```diff
@@ -0,0 +1,23 @@
+import { IReportResultModel } from 'tno-core';
+
+export const openPreviewInNewTab = (previewContent: IReportResultModel) => {
+  const previewHTML = `
+      <html>
+      <head>
+        <title>Report Preview</title>
+      </head>
+      <body>
+        <h1> ${previewContent.subject}</h1>
```

Comment:
maybe just prefix this element with "Report Preview: " like this?
`<h1>Report Preview: ${previewContent.subject}</h1>`
You could do the same thing with the `<title>` as well.